services/user.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
```

Comment:
The import of `User` from `django.contrib.auth.models` is unnecessary since `get_user_model()` is used. Consider removing this import.